download.py

```diff
@@ -212,14 +214,13 @@ def download():
         adjust = 0
 
     if points:
-        points_datasets = [[get_subset(dataset, p, adjust, limit) for dataset in datasets] for p in points]
-
-    if bbox:
+        subsetted_datasets = [[get_subset(dataset, p, adjust, limit) for dataset in datasets] for p in points]
+    else:
```

Comment:
Just a quick question. Since it went from `if bbox:` to `else:`, is there a guaranty that there will always be a bbox (say, default view screen bbox, or whole dataset region) in the event that `points` is null?

Reply:
Changes doesn't show it, but Lines 168 to 188 already guarantee that boox and points are mutually exclusive:

https://github.com/CanadianClimateDataPortal/climatedata-api/blob/b8393f46cacc923769b8e7cc1891bf4a284da37d/download.py#L168